src/txs/stake/QuickStakeForm.tsx

```diff
@@ -139,7 +139,7 @@ const QuickStakeForm = (props: Props) => {
 
   const token = action === QuickStakeAction.DELEGATE ? denom : ""
   const tx = {
-    decimals: readNativeDenom(token)?.decimals,
+    decimals: readNativeDenom(network[chainID].baseAsset).decimals,
```

Comment:
Also, why was this updated from token, but line `183` in `StakeForm.tsx` was left alone?